action-scripts/brownie/scripts/script_utils.py

```diff
@@ -148,16 +169,20 @@ def merge_files(
         "file3.json": "report4",
     }
     """
-    merged_dict = {}
-    for key in added_gauges.keys() | removed_gauges.keys() | transfers.keys() | permissions.keys():
-        merged_dict[key] = "".join([
-            added_gauges.get(key, ""),
-            removed_gauges.get(key, ""),
-            transfers.get(key, ""),
-            permissions.get(key, "")
-        ])
-    return merged_dict
+    strings_by_file = defaultdict(str)
+    for result_output in results_outputs_list:
+        for file, report_data in result_output.items():
+            strings_by_file[file] += report_data["report_text"]
+    return strings_by_file
+
 
+def extract_bip_number_from_file_name(file_name: str) -> str:
```

Comment:
What's the point of this copypasted function? You `extract_bip_number` below that extracts bip number from file?

Reply:
Because in this case I don't have this file: formatted thing.  Or maybe I do and don't realize how to use the other function in my current context.

Reply:
It has same signature, try using `extract_bip_number` ?

Reply:
I think I did.  What should I pass as file to extract_bip_number? It doesn't work as a file_name string.